concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -839,6 +839,34 @@ jobs:
 
                 cat vpc-peering-tfvars/vpc-peers.tfvars
 
+      # FIXME: remove this when it's run everywhere.
+      - task: extract-acm-cert-arn
+        config:
+          platform: linux
+          image_resource: *awscli-image-resource
+          params:
+            AWS_DEFAULT_REGION: ((aws_region))
+          inputs:
+            - name: cf-tfstate
+          outputs:
+            - name: apps-cert
+          run:
+            path: sh
+            args:
+            - -e
+            - -c
+            - |
+              tf_cert_details=$(jq -r '.modules[].resources."aws_acm_certificate.apps"' cf-tfstate/cf.tfstate)
+              if [ "${tf_cert_details}" = "null" ]; then
+                echo "Finding cert to import"
+                aws acm list-certificates \
```

Comment:
Creating a new environment would fail, because the `aws acm list-certificates` returns an empty string if the certificate doesn't exist. We have to handle this case too.

Reply:
Added a check for non-existing cert.